config/config.go

```diff
@@ -85,7 +86,16 @@ func (c Config) CacheFile() string {
 	return path.Join(cacheDir, cacheFileName)
 }
 
+func hasAccess(path string) bool {
+    return unix.Access(path, unix.R_OK) == nil
+}
+
 func checkAndCreateDir(path string) string {
+	isAccsessible := hasAccess(path)
```

Comment:
@Pearl1594 could the fix to be simply to try and open the file and handle the failure? It's because people may use cmk on Mac and Windows.

Reply:
Right! makes sense, it slipped my mind that we support other OSs too.